src/components/button/button.tsx

```diff
@@ -35,6 +35,8 @@ export interface ButtonProps {
     LeftIcon?: React.ElementType;
     RightIcon?: React.ElementType;
     disabled?: boolean;
+    isSubmitButton?: boolean;
```

Comment:
Why not passing the type prop directly?

Reply:
That sounds good too. We're not using this prop yet in Portal, so we can change this and do it that way as soon as we find some time 😄